src/screens/AccountNew.js

```diff
@@ -92,13 +100,13 @@ class AccountNewView extends React.Component {
               <Button
                 buttonStyles={styles.nextStep}
                 title="Next Step"
-                disabled={ !validateSeed(selected.seed, selected.validBip39Seed).valid }
```

Comment:
I don't think it makes sense here to validate the seed since it's not a use input, unlike for the recovery screen.